EventIndex.json

```diff
@@ -3,6 +3,17 @@
     
 "Events": [
 {
+    "Id" : "PROGRAMMING2017",
+    "Name" : "<Programming> 2017",
```

Comment:
@rtholmes if the angle brackets are a problem, please use "‹Programming› 2017" instead.